api/routers/edr.py

```diff
@@ -87,13 +90,17 @@ async def get_data_location_id(
 )
 async def get_data_position(
     coords: str = Query(..., example="POINT(5.179705 52.0988218)"),
-    parameter_name: str = Query(..., alias="parameter-name", example="dd,ff,rh,pp,tn"),
-    datetime: str | None = None,
+    parameter_name: str = Query(None, alias="parameter-name", example="dd,ff,rh,pp,tn"),
+    datetime: str = Query(None, example="2022-12-31T00:00Z/2023-01-01T00:00Z"),
     f: str = Query(default="covjson", alias="f", description="Specify return format."),
 ):
-    point = wkt.loads(coords)
-    assert point.geom_type == "Point"
-    poly = buffer(point, 0.0001, quad_segs=1)  # Roughly 10 meters around the point
+    try:
+        point = wkt.loads(coords)
+        assert point.geom_type == "Point"
```

Comment:
Same as in get_data_area, error should differentiate between error in parsing wkt and providing wrong geometry type.

Reply:
I don't think we want to handle this with an `assert` in production code.

Reply:
Added explicit error messages to parsing & wrong geom types.

Reply:
Changed to raise and catch TypeErrors instead of asserts in https://github.com/EURODEO/e-soh/pull/30/commits/b732cdcc1420137f7882a7c52b10e702eef9781d